File: Java/Empresa.java
import java.util.ArrayList;
import java.util.List;

public class Empresa {

    private String nome;
    private List<Empregado> empregados;

    public Empresa(String nome){
        this.nome = nome;
        this.empregados = new ArrayList<>();
    }

    public void addEmpregado(Empregado empregado){
        empregados.add(empregado);
    }

    public void apresentarEmpregados() {
        System.out.println(nome + " tem os seguintes funcionários: ");
        for (Empregado empregado : empregados) {
            empregado.infoEmpregado();
        }
    }
}

File: Python/Empresa.py
class Empresa:
    def __init__(self, nome):
        self.nome = nome
        self.empregados = []

    def add_empregado(self, empregado):
        self.empregados.append(empregado)

    def apresentar_empregados(self):
        print(f"{self.nome} tem os seguintes funcionários:")
        for empregado in self.empregados:
            empregado.info_empregado()